addons/mail/static/src/js/form_renderer.js

```diff
@@ -42,6 +53,22 @@ FormRenderer.include({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    async _mount() {
+        const props = { id: this.state.res_id, model: this.state.model };
+        if (this.chatter_component) {
+            this.chatter_component.destroy();
+            this.chatter_component = undefined;
+        }
+        this.chatter_component = new OwlChatter(null, props);
+        // FIXME loading a lot of messages that should not
+        // FIXME scroll to top is active
+        // FIXME postMessage in the thread to refresh chatter directly
+        await this.chatter_component.mount(this.$el[0]);
```

Comment:
chatter component should be mounted once, and update props when it exists already